README.md

```diff
@@ -5,31 +5,31 @@
 
 #### Compose a cloud native masterpiece.
 
-Infused with cloud native capabilities from the moment you start, Appsody provides everything you need to iteratively develop applications, ready for deployment to Kubernetes environments. Teams are empowered with sharable technology stacks, configurable and controllable through a central hub.
+Infused with cloud native capabilities from the moment you start, Appsody provides everything you need to iteratively develop applications, ready for deployment to Kubernetes environments. Teams are empowered with sharable technology stacks, configurable and controllable through a central hub. To find out more about Appsody check out [appsody.dev](https://appsody.dev).
```

Comment:
Can you put in a comma after i.e. "To find out more about Appsody, check out ..."